sbpy/photometry/core.py

```diff
@@ -5,13 +5,255 @@
 created on June 23, 2017
 """
 
-__all__ = ['DiskIntegratedModelClass', 'HG', 'HG12', 'HG1G2',
-           'DiskFunctionModel', 'LommelSeeliger', 'Lambert',
+__all__ = ['ref2mag', 'mag2ref', 'spline',
+           'DiskIntegratedModelClass', 'HG', 'HG12', 'HG1G2',
+           'DiskFunctionModel', 'LommelSeeliger', 'Lambert', 'LunarLambert',
            'PhaseFunctionModel', 'ROLOPhase',
            'ResolvedPhotometricModelClass', 'ROLO']
 
+import numpy as np
+from scipy.integrate import quad
+from astropy.modeling import FittableModel, Fittable1DModel, Fittable2DModel, Parameter
+from astropy import units
+
+
+def ref2mag(ref, radius, M_sun=None):
```

Comment:
I personally don't care, but many people prefer not to have free floating functions that are not part of a class. Should we put these functions into a general class, e.g. `Photometry`? That's what astropy does. @migueldvb, any comments on this?

Reply:
I don't understand that perspective, because they are in a sub-module, so still in some kind of namespace.  I don't see the benefit to putting these in a class.

Reply:
In other words, I am interested in hearing why they should be in classes :)

Reply:
... me too :)

Reply:
I thought about including it into the `DiskIntegratedModelClass`, but then considered that they should be general-purpose tools and should not belong to any class but rather as functional tools in the package.  Looks like we've come to an agreement at this point.